tests/test_clauses.py

```diff
@@ -31,3 +32,10 @@ def test_where():
     query, params = render(':v', v=clauses.Where((V('x') == 4) & (V('y').like('xxx'))))
     assert 'WHERE x = $1 AND y LIKE $2' == query
     assert [4, 'xxx'] == params
+
+
+@pytest.mark.parametrize('value,expected', [(0, 0), (False, V('FALSE')), ('', V(''))])
```

Comment:
not sure this is a new problem or has always been there, but surely params shouldn't include `V(...)` objects?

Either `render('WHERE :a', a=V('a') == value)` should return `('WHERE a = FALSE', ())` or `('WHERE a = $1', False)`?

If this isn't a new problem we can like with it here and I'll try to fix it in another PR.

Reply:
So I'm guessing the `V(...)` bit is unnecessary and it's effectively just doing a string equality, so the actual check should be `(False, 'FALSE')` and `('', '')`.

Not sure if it's worth baking the boolean into the statement, there might be a performance argument (?), but I don't know enough about asyncpg's internals to make that call.

Reply:
ignore previous, I will check that asyncpg accepts raw boolean's (I assume it does) and if so then the latter `('WHERE a = $1', False)` sounds correct to me.

Reply:
```◤ (venv) ~ python a.py
[<Record foo='a' bar=True>]
[<Record foo='b' bar=False>]
◤ (venv) ~ cat a.py
import asyncio
import asyncpg

async def run():
    conn = await asyncpg.connect(user='user', password='pass',
                                 database='db', host='127.0.0.1')
    await conn.execute("CREATE TABLE mytable (foo varchar(80), bar boolean)")
    await conn.execute("INSERT INTO mytable (foo, bar) VALUES ('a', $1)", True)
    await conn.execute("INSERT INTO mytable (foo, bar) VALUES ('b', $1)", False)
    values = await conn.fetch("SELECT * FROM mytable WHERE bar = $1", True)
    print(values)
    values = await conn.fetch("SELECT * FROM mytable WHERE bar = $1", False)
    print(values)
    await conn.close()

loop = asyncio.get_event_loop()
loop.run_until_complete(run())```

Will make the changes

Reply:
Right that makes much more sense, The reason it worked before and now is that of course `V` has a custom `__equals__` method that returned another `V` object which is truey.

Hence the very confusion situation where both these lines can pass:

```py
assert params == [False]
assert params == [V('FALSE')]
```